apache/superset
# feat(frontend): add ability to download dashboard and chart as image

### SUMMARY

Adds an additional menu option and functionality to export dashboards and charts as an image.


### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

download dashboard

![download-image-example](https://user-images.githubusercontent.com/3107513/82126452-cf136c80-9761-11ea-9c58-0491b3c7f9b5.gif)

download chart

![download-chart-example](https://user-images.githubusercontent.com/3107513/82127241-d8eb9e80-9766-11ea-9c87-1eac411a9e36.gif)


### TEST PLAN


CI and click testing.

* Pull down branch
* Start superset
* Navigate to a dashboard
* Open menu and select "Download as image"
* After some processing, a prompt to download image should appear
* Image should match dashboard

### ADDITIONAL INFORMATION


- [x] Has associated issue: #6737
